infra/dockerfile.spawn_data_pipeline

```diff
@@ -0,0 +1,11 @@
+#syntax=docker/dockerfile:experimental
```

Comment:
We don't need the experimental here, right?

Reply:
Yeap: there are no `RUN --mount` commands